2022-07-06 15:28:45.688 [WRN] {"reply":{"restaurants":[{"city":"Hà Nội","foody_service":"FOOD","name":"Phúc Long Hà Đông 0622","restaurant_id":1000007107,"address":"8 Quang Trung","partner_restaurant_id":"2001"}]},"result":"success"}
2022-07-06 15:51:04.371 [WRN] {"reply":{"restaurants":[{"city":"Hà Nội","foody_service":"FOOD","name":"Phúc Long Hà Đông 0622","restaurant_id":1000007107,"address":"8 Quang Trung","partner_restaurant_id":"2001"}]},"result":"success"}
2022-07-06 15:51:05.721 [ERR] IPhucLongShopeeService.InsertShopeeRestaurant.Exception Cannot access a disposed object. A common cause of this error is disposing a context that was resolved from dependency injection and then later trying to use the same context instance elsewhere in your application. This may occur if you are calling Dispose() on the context, or wrapping the context in a using statement. If you are using dependency injection, you should let the dependency injection container take care of disposing context instances.
Object name: 'SalesPartnerDbContext'.
2022-07-06 15:51:05.751 [ERR] Connection id "0HMIV5IHTE291", Request id "0HMIV5IHTE291:00000009": An unhandled exception was thrown by the application.
System.InvalidOperationException: The connection does not support MultipleActiveResultSets.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.ValidateConnectionForExecute(SqlCommand command)
   at Microsoft.Data.SqlClient.SqlInternalTransaction.Rollback()
   at Microsoft.Data.SqlClient.SqlInternalTransaction.Dispose(Boolean disposing)
   at Microsoft.Data.SqlClient.SqlInternalTransaction.Dispose()
   at Microsoft.Data.SqlClient.SqlTransaction.Dispose(Boolean disposing)
   at System.Data.Common.DbTransaction.DisposeAsync()
   at Microsoft.EntityFrameworkCore.Utilities.DisposableExtensions.DisposeAsyncIfAvailable(IDisposable disposable)
   at Microsoft.EntityFrameworkCore.Storage.RelationalTransaction.DisposeAsync()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.DisposeAsync()
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.<>c__DisplayClass15_0.<<DisposeAsync>g__Await|0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.<>c__DisplayClass15_0.<<DisposeAsync>g__Await|0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Http.Features.RequestServicesFeature.<DisposeAsync>g__Awaited|9_0(RequestServicesFeature servicesFeature, ValueTask vt)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.FireOnCompletedAwaited(Task currentTask, Stack`1 onCompleted)
2022-07-06 15:54:55.252 [WRN] {"reply":{"restaurants":[{"city":"Hà Nội","foody_service":"FOOD","name":"Phúc Long Hà Đông 0622","restaurant_id":1000007107,"address":"8 Quang Trung","partner_restaurant_id":"2001"}]},"result":"success"}
2022-07-06 15:55:00.519 [WRN] {"reply":{"restaurants":[{"city":"Hà Nội","foody_service":"FOOD","name":"Phúc Long Hà Đông 0622","restaurant_id":1000007107,"address":"8 Quang Trung","partner_restaurant_id":"2001"}]},"result":"success"}
2022-07-06 15:55:04.467 [WRN] {"reply":{"restaurants":[{"city":"Hà Nội","foody_service":"FOOD","name":"Phúc Long Hà Đông 0622","restaurant_id":1000007107,"address":"8 Quang Trung","partner_restaurant_id":"2001"}]},"result":"success"}
2022-07-06 15:55:37.572 [WRN] {"reply":{"restaurants":[{"city":"Hà Nội","foody_service":"FOOD","name":"Phúc Long Hà Đông 0622","restaurant_id":1000007107,"address":"8 Quang Trung","partner_restaurant_id":"2001"}]},"result":"success"}
2022-07-06 15:57:00.195 [WRN] {"reply":null,"result":"error_invalid_res_id"}
2022-07-06 15:57:59.526 [WRN] {"reply":{"restaurants":[{"city":"Hà Nội","foody_service":"FOOD","name":"Phúc Long Hà Đông 0622","restaurant_id":1000007107,"address":"8 Quang Trung","partner_restaurant_id":"2001"}]},"result":"success"}
2022-07-06 15:58:04.990 [WRN] {"reply":{"restaurants":[{"city":"TP. HCM","foody_service":"FOOD","name":" Phúc Long - Huỳnh Thúc Kháng","restaurant_id":29971,"address":"26 Huỳnh Thúc Kháng","partner_restaurant_id":"2002"}]},"result":"success"}
2022-07-06 15:58:05.196 [ERR] Connection id "0HMIV5ME16L0Q", Request id "0HMIV5ME16L0Q:0000000B": An unhandled exception was thrown by the application.
System.InvalidOperationException: The connection does not support MultipleActiveResultSets.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.ValidateConnectionForExecute(SqlCommand command)
   at Microsoft.Data.SqlClient.SqlInternalTransaction.Rollback()
   at Microsoft.Data.SqlClient.SqlInternalTransaction.Dispose(Boolean disposing)
   at Microsoft.Data.SqlClient.SqlInternalTransaction.Dispose()
   at Microsoft.Data.SqlClient.SqlTransaction.Dispose(Boolean disposing)
   at System.Data.Common.DbTransaction.DisposeAsync()
   at Microsoft.EntityFrameworkCore.Utilities.DisposableExtensions.DisposeAsyncIfAvailable(IDisposable disposable)
   at Microsoft.EntityFrameworkCore.Storage.RelationalTransaction.DisposeAsync()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.DisposeAsync()
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.<>c__DisplayClass15_0.<<DisposeAsync>g__Await|0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.<>c__DisplayClass15_0.<<DisposeAsync>g__Await|0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Http.Features.RequestServicesFeature.<DisposeAsync>g__Awaited|9_0(RequestServicesFeature servicesFeature, ValueTask vt)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.FireOnCompletedAwaited(Task currentTask, Stack`1 onCompleted)
2022-07-06 15:58:05.273 [ERR] IPhucLongShopeeService.InsertShopeeRestaurant.Exception Cannot access a disposed object. A common cause of this error is disposing a context that was resolved from dependency injection and then later trying to use the same context instance elsewhere in your application. This may occur if you are calling Dispose() on the context, or wrapping the context in a using statement. If you are using dependency injection, you should let the dependency injection container take care of disposing context instances.
Object name: 'SalesPartnerDbContext'.
2022-07-06 15:58:16.177 [WRN] {"result":"success","reply":{"dishes":[{"picture":{"url":"http://media-qa.foody.vn/res/g100001/1000007107/s180x180/37a15c0e-756f-4a68-8be9-ac9e2869-bd1c44bf-220628132814.jpeg","id":712662},"partner_dish_id":"60000074","description":"","price":35000.0,"is_active":true,"dish_group_id":369424,"created_time":"2022-06-28 13:28:15","name_en":"","name":"Trà vải","display_order":0,"updated_time":"2022-06-30 16:45:40","dish_id":10378623,"partner_dish_group_id":"1"},{"picture":{"url":"http://media-qa.foody.vn/res/g100001/1000007107/s180x180/c2ffbc2c-e3dc-4791-87ea-982267d4-ca8e3044-220628132838.jpeg","id":712663},"partner_dish_id":"60000075","description":"","price":40000.0,"is_active":true,"dish_group_id":369424,"created_time":"2022-06-28 13:28:38","name_en":"","name":"Cafe sữa","display_order":0,"updated_time":"2022-06-30 17:54:48","dish_id":10378624,"partner_dish_group_id":"1"}]}
